### DISCUSSION TITLE
Can't change repository URL myself

### DISCUSSION THREAD

**Discussion by deliqs** [codemagic-ci-cd/codemagic-docs#1920] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I would like to change my repository URL as I have merged a few applications in a monorepo. But it seems I am not able to do this myself.

App URL is https://codemagic.io/app/617e386b54ed64d0daf1f6a7
The current URL is https://github.com/deliqs/deliqs_app
The new URL is https://github.com/deliqs/deliqs_platform

Could somebody change this for me?

### Expected results

Repository URL is editable

### Actual results

Repository URL is readonly

**Comment by kulakarla** [comment]
Hi @deliqs 

Yes, the change is possible.

This is how we should proceed:
1) Add the new monorepo as an Application to your Codemagic account, but leave the old application tied to deliqs_app repository as is, don't delete it.
2) Provide me the app id-s of the freshly added monorepo application and the old deliqs_app repository application. Make sure to note which is which. App id can be retrieved from the URL following ``/app/`` when opening an app: ``https://codemagic.io/app/<appId>``.
3) Having given me the id-s specified with which is the monorepo application and which is the old application that you'd like to update, I will do the change for you.

**Reply by deliqs** [reply]
So you move all the settings from the old app to the new app?

Old app ID = 617e386b54ed64d0daf1f6a7
New app ID = 64c2571062ea7f5384ff1908

**Reply by kulakarla** [reply]
I will change the repo URL of the old app to the new app. That way all the settings, environment variables etc stay in place.

**Reply by kulakarla** [reply, marked as answer]
@deliqs 

The change should be made. You can delete the freshly added monorepo application, as your old applications repository URL should now be the monorepo URL. Ofcourse, before you delete anything, double check that the change has been successful.

**Reply by deliqs** [reply]
Yes, perfect! Thanks @kulakarla

**Comment by Moh-Afifi-1024** [comment]
Hey @kulakarla  I have the same issue and I need your support please.
I have two applications and I need to migrate from bitbucket to github
First App:
Old App Id: 63dad5358d8ae43d9774f9b1
New App Id: 67f29b71ffc2812baefa7837

Second App:
Old App Id: 63e150db94a6e72db44e1892
New App Id: 67f29b8b92bbe035290eebde

**Reply by kulakarla** [reply]
Hey, this is a closed issue and I don't work there anymore. Perhaps you can open up a new thread?
